### DISCUSSION TITLE
Janus + Elassandra

### DISCUSSION THREAD

**Discussion by dioptre** [JanusGraph/janusgraph#2186] [Ideas]
This is less of a feature request. Just wanted you to know, I prefer working with Elassandra, so made a docker image to get started with Janus + Elassandra.

https://github.com/sfproductlabs/jelass

For those that don't know of it, it's great and have been using it in production for years. If someone were interested I could add some docs to the wiki about it.

**Comment by FlorianHockmann** [comment]
Hi @dioptre and thanks for letting us know about this project! If you want, you can also announce it on our [janusgraph-users Google group](https://groups.google.com/g/janusgraph-users) as you probably reach a wider audience there. Maybe also add the key advantages / disadvantages of this setup (especially compared to just using Cassandra and ES) as many people probably don't know about Elassandra and therefore also don't know why they should use it. I at least haven't heard about this before so that information would be useful to me.

If there is substantial interest by users, then we can of course also add something about his project to the docs.

**Comment by dioptre** [comment]
Ok thanks, I added it to the group @FlorianHockmann - not sure if it needs to be approved - let me know

**Comment by dioptre** [comment]
FYI seems like there's some small de-duplication necessary (to subtract from the existing cql-es example) for a really nice fit long term.  https://github.com/strapdata/elassandra/issues/366

**Comment by dioptre** [comment]
Not sure if my post needs to get approved on the list @FlorianHockmann ?

**Comment by FlorianHockmann** [comment]
Yes, it does need to be improved which can take a bit of time. That should however only be necessary for your first post in the group.
